dgraph/cmd/bulk/run.go

```diff
@@ -158,10 +158,6 @@ func run() {
 	if opt.Version {
 		os.Exit(0)
 	}
-	if opt.BadgerCompressionLevel < 0 {
```

Comment:
This has been removed because this is derived from `x.ParseCompression`, which calls `glog.Fatal` in case of a negative compression level. Thus, there is no need to check over here.